server/routes/findInterventions/InterventionDetailsPresenter.ts

```diff
@@ -0,0 +1,82 @@
+import { Eligibility, Intervention } from '../../services/interventionsService'
+import { SummaryListItem } from '../../utils/summaryList'
+import utils from '../../utils/utils'
+
+export interface InterventionDetails {
```

Comment:
What is the intention of this type? Would it be a problem for `SearchResultsPresenter`'s `results` method to return `InterventionDetailsPresenter[]`?

Reply:
Yeah I think this is due to the first way I implemented this (badly) then I refactored it and must have forgotten to update this - well spotted.